schema.json

```diff
@@ -1165,7 +1165,7 @@
             "type": "string"
         },
         "person": {
-            "additionalProperties": false,
+            "additionalProperties": true,
```

Comment:
Note to self: this change probably requires adding `additionalProperties: false` to usages of the definition, otherwise you can for example add props (or misspell props) to an author.